src/components/molecules/WorkspaceEntry.tsx

```diff
@@ -0,0 +1,83 @@
+import { WorkspaceMetadata } from '../../types/application-data/application-data';
+import { Box, Button, Card, CardContent, Typography } from '@mui/joy';
+import { TextAvatar } from '../atoms/TextAvatar';
+import { SprocketTooltip } from '../atoms/SprocketTooltip';
+import { formatDate } from '../../utils/string';
+import EditCalendarIcon from '@mui/icons-material/EditCalendar';
+import InfoIcon from '@mui/icons-material/Info';
+import OpenInNewIcon from '@mui/icons-material/OpenInNew';
+import DeleteIcon from '@mui/icons-material/Delete';
+import { setWorkspace } from '../../state/workspaces/slice';
+import { useAppDispatch } from '../../state/store';
+
+interface WorkspaceEntryProps {
+	workspace: WorkspaceMetadata;
+	onDelete: (workspace: WorkspaceMetadata) => void;
+}
+
+export function WorkspaceEntry({ workspace, onDelete }: WorkspaceEntryProps) {
```

Comment:
Split this out because I felt it could be. named it "Entry" because I didn't want to ascribe to an only-card view going forward - who knows, this might hold rows as well as cards, yeah?